huaweicloud/services/cbr/resource_huaweicloud_cbr_vault.go

```diff
@@ -245,72 +255,113 @@ func buildDissociateResourcesForDisk(rType string, resources []interface{}) []st
 func buildDissociateResources(vType string, resources *schema.Set) ([]string, error) {
 	rType, ok := ResourceType[vType]
 	if !ok {
-		return nil, fmtp.Errorf("Invalid resource type: %s", vType)
+		return nil, fmt.Errorf("invalid resource type: %s", vType)
 	}
-	logp.Printf("[DEBUG] The resource type is %s", rType)
+	log.Printf("[DEBUG] The resource type is %s", rType)
 	switch rType {
 	case ResourceTypeServer:
 		return buildDissociateResourcesForServer(rType, resources.List()), nil
 	case ResourceTypeDisk, ResourceTypeTurbo:
 		return buildDissociateResourcesForDisk(rType, resources.List()), nil
 	default:
-		return nil, fmtp.Errorf("The vault type only support server, disk and turbo.")
+		return nil, fmt.Errorf("the vault type only support server, disk and turbo")
 	}
 }
 
-func buildCBRVaultBilling(d *schema.ResourceData) *vaults.BillingCreate {
+func isPrePaid(d *schema.ResourceData) bool {
+	if val, ok := d.GetOk("charging_mode"); ok && val.(string) == "prePaid" {
+		return true
+	}
+	return false
+}
```

Comment:
This func equals to `if d.Get("charging_mode").(string) == "prePaid"`. Why we need it?

Reply:
done.